static/src/javascripts/projects/common/modules/onward/popular.js

```diff
@@ -44,7 +46,11 @@ define([
     };
 
     MostPopular.prototype.prerender = function () {
-        if (!(config.page.shouldHideAdverts || this.mobileMaximumSlotsReached())) {
+        if (!(
+            config.page.shouldHideAdverts || /* sensitive pages */
```

Comment:
Yeah, I totally get that, no doubt there's plenty of permutations to consider.

> I did think about perhaps creating a map of flags that would define which features would and wouldn't be turned on, and would let us define ad-free and sensitive-mode behaviour in fairly self-documenting ways,

Would definitely like to see this, but agree it might not be needed yet; I don't know how permanent this feature is or if it's just a test for now.